StateMachine/src/main.cpp

```diff
@@ -0,0 +1,85 @@
+#include <Arduino.h>
+#include <Wire.h>
+#include <SPI.h>
+#include <FlexCAN_T4.h>
+
+#include "ChRt.h"
```

Comment:
If threading proves to be error-prone we might remove it and try again in the future if we see the need. Feel free to leave it in for now